src/Microsoft.AspNet.Authentication/AuthenticationServiceCollectionExtensions.cs

```diff
@@ -12,6 +12,7 @@ public static class AuthenticationServiceCollectionExtensions
     {
         public static IServiceCollection AddAuthentication([NotNull] this IServiceCollection services)
         {
+            services.AddCaching();
```

Comment:
It really sounds like a PITA for almost no benefits, as the memory caching package that offers the `services.AddCaching()` extension doesn't depend on packages that are not referenced by the other security packages.

But meh, fixed anyway.